Next.js instructions for RUM (#19360)
DataDog/documentation: content/en/real_user_monitoring/connect_rum_and_traces/_index.md

RUM supports several propagator types to connect resources with backends that are instrumented with OpenTelemetry libraries.

{{< tabs >}}
{{% tab "Browser RUM" %}}

> **Note**: If you are using a backend framework such as Next.js/Vercel that is tracing with OpenTelemetry, you need to follow these steps.

Reviewer: ```suggestion
**Note**: If you are using a backend framework such as Next.js/Vercel that uses OpenTelemetry, follow these steps.
```
